replications/metrics/replications_metrics.go

```diff
@@ -0,0 +1,59 @@
+package metrics
+
+import (
+	"github.com/influxdata/influxdb/v2/kit/platform"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type ReplicationsMetrics struct {
+	TotalPointsQueued  *prometheus.CounterVec
+	TotalBytesQueued   *prometheus.CounterVec
+	CurrentBytesQueued *prometheus.GaugeVec
+}
+
+func NewReplicationsMetrics() *ReplicationsMetrics {
+	const namespace = "replications"
+	const subsystem = "queue_management"
```

Comment:
Self-review: These metric names end up being really long - evaluate it with a shorter subsystem name